_includes/v22.2/sql/privileges.md

```diff
@@ -1,18 +1,18 @@
 Privilege | Levels
 ----------|------------
-`ALL` | Database, Schema, Table, Type
-`CREATE` | Database, Schema, Table
-`DROP` | Database, Table
+`ALL` | Database, Schema, Table, Sequence, Type
+`CREATE` | Database, Schema, Table, Sequence
+`DROP` | Database, Table, Sequence
 `EXECUTE` | Function
-`GRANT` | Database, Function, Schema, Table, Type
+`GRANT` | Database, Function, Schema, Table, Sequence, Type
 `CONNECT` | Database
-`SELECT` | Table
-`INSERT` | Table
-`DELETE` | Table
-`UPDATE` | Table
-`USAGE`  | Function, Schema, Type
-`ZONECONFIG` | Database, Table
+`SELECT` | Table, Sequence
+`INSERT` | Table, Sequence
+`DELETE` | Table, Sequence
+`UPDATE` | Table, Sequence
+`USAGE`  | Function, Schema, Sequence, Type
+`ZONECONFIG` | Database, Table, Sequence
 `EXTERNALCONNECTION` | System
-`BACKUP` | System, Database, Table
+`BACKUP` | System, Database, Table, Sequence
```

Comment:
> For the BACKUP privilege, I think we should remove sequence from this row.

I agree. We actually don't have a `BACKUP SEQUENCE` syntax support. 

User can use `BACKUP TABLE seq_name` to backup a sequence, but again, this is because sequence has been treated as a special subcategory of tables. To be align with what we have for `GRANT...ALL TABLES IN SCHEMA`, i think we should remove sequence here for backup.